backend/LeetWars.RabbitMQ/Services/NotifierCodeConsumerService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+
+namespace LeetWars.RabbitMQ.Services;
+public class NotifierCodeConsumerService : IConsumerService
+{
+    private readonly IConnection _connection;
+    private readonly IModel _channel;
+    private readonly ConsumerSettings _settings;
+    private bool disposedValue;
+
+    public NotifierCodeConsumerService(IConnection connection, ConsumerSettings settings)
```

Comment:
what is the difference between this and ConsumerService?

Reply:
they are the same, but as you suggested we need another consumer service for every queue and because our notifier service in future will have another queue with core this service was created, because as far as we know you can't inject two separate instances of the same interface with the same class that inherits from it